pkg/inferenceservice-controller/suite_test.go

```diff
@@ -177,6 +180,9 @@ var _ = AfterSuite(func() {
 func ModelRegistryDefaultMockServer() *httptest.Server {
 	handler := http.NewServeMux()
 
+	servingEnvironments := new(sync.Map)
+	inferenceServices := new(sync.Map)
```

Comment:
Maybe I missed it, but it doesn't look like these sync maps are needed, does it?

Reply:
I was trying them while debugging a weird bug in tests and I forgot to switch back to regular maps, good catch reverted in https://github.com/kubeflow/model-registry/pull/960/commits/5847b3c5aa7744781107557bad05390d501deaa9

Reply:
If I'm not mistaken, there is only one hardcoded id `1` being used. If so, does it even need a map?

Reply:
Not for the time being, I'm going to keep it in case of more complex scenarios